### ISSUE TITLE
How to reset the component?

### ISSUE THREAD

**Issue by fernavat** [PygmySlowLoris/vue-stepper#21]
I run the component in a modal, and when I relaunch the modal after finished the stepper, it shows the last step, so how should I reset it?

**Comment by Edujugon** [comment]
Hi @fernavat ,

We've just released a new package version (1.4.0) that covers the requested functionality.

So now you have available one new prop called `reset` which when it's `true` it will reset the steps of the component. After doing so it will emit an event called `reset` where you can set the mentioned prop again to `false`.

```javascript
<horizontal-stepper 
  :reset="reset"
  @reset="resetEvent"
  ....
></horizontal-stepper>
```

Hope this helps.

**Comment by fernavat** [comment]
Excellent, thanks!
